cmake/maybe_build_using_clangw.cmake

```diff
@@ -48,31 +44,30 @@ function(maybe_build_using_clangw OE_TARGET)
 
     # Setup library tool variables
     set(CMAKE_C_CREATE_STATIC_LIBRARY 
-        "\"${CMAKE_BINARY_DIR}/windows/clangw/llvm-arw.exe\" qc <TARGET> <OBJECTS>" 
+        "bash \"${PROJECT_SOURCE_DIR}/scripts/llvm-arw\" \"qc <TARGET> <OBJECTS>\"" 
```

Comment:
We should probably be using `${BASH}` (already set in another file from `find_program` call, and we might need to move that upwards).